### PR TITLE
upgrade rust to 1.71.0

### PR DESCRIPTION
**Pull request by dead10ck**
Rust is currently at v1.72.0. We have to upgrade to 1.71.1 first to get to 1.72.0

### CODE REVIEW

**Comment by dead10ck** [comment]
Looks like I got it passing! Though I'm noticing the "Generate build artifacts" step has some messages about missing files. Is this expected?

**Comment by truboxl** [comment]
Seems like this missing files issue already happened before, will keep it mind for the next rewrite of rust build.sh
https://github.com/termux/termux-packages/actions/runs/5438550048/job/14722252430#step:6:6755

File: packages/rust/build.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.rust-lang.org/
 TERMUX_PKG_DESCRIPTION="Systems programming language focused on safety, speed and concurrency"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.70.0
+TERMUX_PKG_VERSION=1.71.1
 TERMUX_PKG_REVISION=1
```
Comment (truboxl) [inline packages/rust/build.sh:6]:
```suggestion
```
Should remove TERMUX_PKG_REVISION when bumping TERMUX_PKG_VERSION

**Review by truboxl — APPROVED** [review]
Thanks. Will help squash and edit the commit title.

**Pull request merged**